tests/test_proxy_functional.py

```diff
@@ -838,6 +839,21 @@ async def xtest_proxy_from_env_https_with_auth(
     assert r2.headers["Proxy-Authorization"] == auth.encode()
 
 
+@pytest.mark.xfail
+async def xtest_proxy_https_proxy_headers(proxy_test_server: Any, loop: Any) -> None:
```

Comment:
@Dreamsorcerer @galaxyfeeder this also shouldn't start with `xtest` because this hides the function from the pytest collection process.
```suggestion
async def test_proxy_https_proxy_headers(proxy_test_server: Any, loop: Any) -> None:
```